apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -17,6 +17,8 @@ const { normalizeEmail } = require('@condo/domains/common/utils/mail')
 const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@condo/keystone/errors')
 const { WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
 
+const SUPPORT_EMAIL_MOBILE = conf['HELP_REQUISITES'] && get(JSON.parse(conf['HELP_REQUISITES']), 'support_email_mobile', null)
+if (!SUPPORT_EMAIL_MOBILE) throw new Error('support_email_mobile variable not found. help_requisites must be specified in .env')
```

Comment:
But what about this comment?

 >  Fix .env before merging. Support creds must be optional! Service must work without them